base/ApplicationSupport.F90

```diff
@@ -31,8 +31,7 @@ subroutine MAPL_Initialize(unusable,comm,logging_config,rc)
          logging_configuration_file=''
       end if
       if (present(comm)) then
-         call MPI_comm_dup(comm,comm_world,status)
-         _VERIFY(status)
+         comm_world = comm
```

Comment:
How do we know it is safe not to dup the communicator here?

Reply:
It should be safe if  line 36 comm_world=MPI_COMM_WORLD is safe

Reply:
I disagree with the reasoning.   In the MPI_COMM_WORLD case, GEOS is not being embedded inside of someone else's application.   (Or if it is, they've already not done anything to insulate themselves.)      Some of GEOS comms in MAPL use TAGs that could potentially conflict with TAGs of the external application.   Hard to imagine any problems in practice, but that is the main reason for DUP in the first place.     Usually more of a concern when going back and forth between two layers.

I suppose we could accept and just  reverse when an issue is found later.